session1/react16/src/js/components/presentational/Fragments.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
great catch! Yeah, React 16.2 added this component to avoid the use of keys, like  it happens when returning an array